UE/Application/States/ConnectedState.hpp

```diff
@@ -15,11 +15,11 @@ class ConnectedState : public BaseState
     void handleSms(const Sms& sms) final;
     void handleShowSmsList() final;
     void handleShowSms(IUeGui::IListViewMode::Selection) final;
-    void handleCallRequest(common::PhoneNumber from) final;
+    void handleReceiveCallRequest(common::PhoneNumber from) final;
 
     // IUserEventsHandler interface
-    void handleCallAccept(common::PhoneNumber to) final;
-    void handleCallDrop(common::PhoneNumber to) final;
+    void handleSendCallAccept(common::PhoneNumber to) final;
+    void handleSendCallDrop(common::PhoneNumber to) final;
```

Comment:
forgot to override "handleStartDial", "handleSendCallRequest" and "handleReceiveCallAccept".